### PR TITLE
Swap lat/lon for proj6 and gdal3.

### PR DESCRIPTION
**Pull request by hardreddata**
Following https://github.com/isce-framework/isce2/issues/278

### CODE REVIEW

**Comment by piyushrpt** [comment]
I have no idea about number of users who are still stuck with GDAL 2. Would it be possible to check gdal version with `gdal.__version__` and handle both?

**Comment by bjmarfito** [comment]
@RussellGrew, my proposed changes to your PR based from the comment of @piyushrpt.

```
if gdal.__version__[0] >= 3:
    ring.AddPoint(earlyNear[0], earlyNear[1])
    ring.AddPoint(earlyFar[0], earlyFar[1])
    ring.AddPoint(lateFar[0], lateFar[1])
    ring.AddPoint(lateNear[0], lateNear[1])
    ring.AddPoint(earlyNear[0], earlyNear[1])
else:
    ring.AddPoint(earlyNear[1], earlyNear[0])
    ring.AddPoint(earlyFar[1], earlyFar[0])
    ring.AddPoint(lateFar[1], lateFar[0])
    ring.AddPoint(lateNear[1], lateNear[0])
    ring.AddPoint(earlyNear[1], earlyNear[0])
```

**Comment by RussellGrew** [comment]
Thanks @bjmarfito 

Time got in the way.

**Review by piyushrpt — APPROVED** [review]

File: contrib/stack/topsStack/plotBursts.py [hunk 1]
```diff
@@ -138,11 +138,18 @@ def main(iargs=None):
                 lateNear = burst.orbit.rdr2geo(t1,r0)
 
                 ring = ogr.Geometry(ogr.wkbLinearRing)
-                ring.AddPoint(earlyNear[1], earlyNear[0])
-                ring.AddPoint(earlyFar[1], earlyFar[0])
-                ring.AddPoint(lateFar[1], lateFar[0])
-                ring.AddPoint(lateNear[1], lateNear[0])
-                ring.AddPoint(earlyNear[1], earlyNear[0])
+                if int(gdal.__version__[0]) >= 3:
```
Comment (rtburns-jpl) [inline contrib/stack/topsStack/plotBursts.py:141]: Looking at the first character of the version string seems potentially brittle. Maybe we can use distutils for a more robust version comparison?
```suggestion
                from distutils.version import StrictVersion
                if StrictVersion(gdal.__version__) >= StrictVersion("3.0")
```

**Pull request merged**